fixtures/dom/src/components/TestCase.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const propTypes = {
+  children: React.PropTypes.node.isRequired,
+  title: React.PropTypes.node.isRequired,
+};
+
+class TestCase extends React.Component {
+
+  render() {
+    const { title, description, children } = this.props;
+
+    return (
+      <section className="test-case">
+        <h2 className="type-subheading">
+          <label>
+            <input type='checkbox' />
```

Comment:
Maybe we can give the fixture test case a slight but noticeable green border when its checked off? That way it's very easy to scroll through quickly and see which tests you haven't done.